perun-cli/anonymizeUser

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+
+sub help {
+	return qq{
+	Anonymizes a user. User id is required.
+	--------------------------------------
+	Available options:
+	--userId   | -u user id
```

Comment:
Please align the indentation with the other options.

Reply:
aligned